src/styles/blocks/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$transition-catalog: color 300ms, transform 300ms;
```

Comment:
I guess there is no need to create a variable for the whole property. 
It would be better to create a variable only for transition-duration, as it is frequently used